examples/winrm/sql_server.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Domotz Custom Driver 
+ * Name: SQLServer Status Monitoring
+ * Description: This script is used to monitor the status of a SQL Server database to check if it is able to serve requests
+ * 
+ * Communication protocol is WinRM
+ * 
+ * Tested on Windows Version
+ *  - Windows 11
+ * 
+ * Powershell Version:
+ *  - 5.1.21996.1
+ * 
+ * Creates a Custom Driver Variable with the status of the SQLServer database
+ * 
+ * Privilege required: Administrator
+ * 
+**/
+
+//The name of the SQL server instance
+var serverName = D.getParameter('serverName');
+//The name of the database
+var databaseName = D.getParameter('databaseName');
+
+// Commands to be executed
+var serviceStatus = "Get-Service 'MSSQL$SQLEXPRESS' | Select-Object Status";
```

Comment:
does this driver support other version of MSSQL servers other thank express. I propose to set the name of MSSQL as a parameter and document the way we could find it